### ISSUE TITLE
Cannot Install Mitaka r13.1.2

### ISSUE THREAD

**Issue by txgunslinger77** [rcbops/rpc-openstack#1674]
##### On which version(s) do you encounter this issue? 
Mitaka r13.1.2

###### Please provide the output to 'git describe --all' for  all deployments where you have confirmed it occurs.
13.3.8-1-g14762c3

##### What type of issue is this? Please tick the appropriate box.
- [X ] bug - existing functionality that does not work for you as documented

##### What is the expected behavior?
setup-infrastructure.yml completes without error and OpenStack is installed

##### What is actually happening?
setup-infrastructure.yml fails to complete. Specifically, the play fails at the repo_build.yml task during creation of the OpenStack-Ansible requirement wheels

##### What were you attempting to do when you encountered this error (give as much detail as possible)? 
Attempting to install Mitaka r13.1.2 at the point where it was time to run the setup-infrastructure.yml play after following these steps:

cd /opt/rpc-openstack/openstack-ansible/playbooks
openstack-ansible setup-hosts.yml
cd ../../rpcd/playbooks ; openstack-ansible pip-lockdown.yml
cd - ; openstack-ansible setup-infrastructure.yml

##### How can we reproduce this issue?
After git cloning Mitaka 13.1.2, bootstrap the deployment host and then proceed to run either deploy.sh or the individual plays until you reach setup-infrastructure.yml

##### How can we verify your issue is addressed?
Mitaka is successfully installed and no errors are encountered during the various plays

##### If you are using or know of a workaround please describe the necessary steps?
Edit /opt/rpc-openstack/rpcd/playbooks/roles/horizon_extensions/defaults/main.yml and change the following variable from 'master' to "mitaka-13.1" like this:
`horizon_extensions_git_install_branch: "mitaka-13.1"`

##### What is your evaluation on the importance of this issue, and why?

Critical, self-explanatory
----------------------------------------------------------------------------------------------------------------------------
###### For each pull request, update the branch line to reference it.
- [ ] master
- [ ] mitaka-13.1
- [ ] liberty-12.2
- [ ] kilo

##### Upgrade impact. Consider the impact on moving between major, minor and patch releases.
- [ ] This change does not impact upgrades
- [ ] This change addresses a potential upgrade impact
- [ ] The upgrade impact is being tracked by another issue.

The issue tracking the upgrade impact is:

##### Testing impact
- [ ] An issue/pull-request has been create or exists to track the testing gap, and is referenced below
- [ ] This change includes additional testing
- [ ] This change is tested by the existing setup

Additional testing links:

##### Release note
- [ ] This change includes a release note.
- [ ] This change does not require a release note.

##### Does your change require documentation?
- [ ] Yes, and I submitted an issue to [Docs issues](https://github.com/rackerlabs/docs-rpc/issues "Docs issues")
- [ ] Yes, and I contributed a documentation patch to [docs-rpc](https://github.com/rackerlabs/docs-rpc "docs-rpc")
- [ ] No documentation required for this change

Additional documentation links (please add links if you have ticked yes):

**Comment by alextricity25** [comment]
This is my fault. The problem was fixed in **https://github.com/rcbops/rpc-openstack/pull/1658**, I just forgot that I needed to create a release after the fix was made.
I'll create a release soon.

**Comment by alextricity25** [comment]
Blocked by https://github.com/rcbops/rpc-openstack/pull/1679

**Comment by alextricity25** [comment]
Unblocked now

**Comment by alextricity25** [comment]
Blocked again by: https://github.com/rcbops/rpc-openstack/issues/1663

**Comment by alextricity25** [comment]
Please use the latest release, r13.1.3. This problem is fixed in that tag.
https://github.com/rcbops/rpc-openstack/releases/tag/r13.1.3
